apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -58,14 +58,14 @@ Page {
                 EnterKey.onClicked: focus = false
             }
 
-            ComboBox {
-                id: searchEngine
+            BrowserListItem {
```

Comment:
No, this will not work as ComboBox handles 5+ items cases in better fashion. Meaning that when there are more items a dedicated page is pushed to the pagestack. I have for instance 16 search engines and expanding context menu for those is no go.

Reply:
So, return ComboBox and... see below.

Reply:
@rainemak  Done.